base/docs/helpdb/Base.jl

```diff
@@ -2603,7 +2603,8 @@ creates a task, and does not run it.
 """
     fld(x, y)
 
-Largest integer less than or equal to `x/y`.
+Largest integer `n` such that `y * n` is less than or equal to `x`. Note that this can be
```

Comment:
So is this the current definition, or was @yuyichao saying we'd need to change the code to make the implementation match this behavior in https://github.com/JuliaLang/julia/issues/17415#issuecomment-232768152?